src/cljc/quil/core.cljc

```diff
@@ -4143,8 +4148,8 @@
   sketches and PDF output in cases where the vector data is available:
   when the font is still installed, or the font is created via the
   create-font fn"
-  ([^PFont font] (.textFont (current-graphics) font))
-  ([^PFont font size] (.textFont (current-graphics) font (int size))))
+  ([font] (.textFont (current-graphics) font))
+  ([font size] (.textFont (current-graphics) font (int size))))
```

Comment:
Why remove ^PFont? These are needed for clj to tell jvm which type is it so it doesn't do unnecessary reflection checks.

Reply:
Yes, it's not performance critical, but I want to keep core.cljc without any reflection warnings. It's simpler to have no warnings then remember which are performance non-critical and which one need to be fixed. 

As for type hints in cljs, I dno't think they mean anything there? At least according to this: https://github.com/clojure/clojurescript-site/blob/master/content/about/differences.adoc#hinting So keeping ^PFont even for cljs should not affect anything. Or you saw some issues with that?

Reply:
Didn't know it was ok to leave the type hint for cljs and while adapting the snippet I was under the impression that leaving it resulted in issues. Modified the PR to only contain the change required for the `text-font` snippet. Sorry for the confusion here.